### DISCUSSION TITLE
dark theme

### DISCUSSION THREAD

**Discussion by sskit** [runew0lf/RuinedFooocus#153] [General]
Hello! 
Can you please tell me how to enable the dark theme?

**Comment by yownas** [comment]
Gradio (the library that build the webui) should take the theme settings from your browser. But you should be able to override it by using this url when accessing the webui: http://localhost:7860/?__theme=dark

**Comment by Ckrotky** [comment]
You can use parameter 

> --theme dark 

for example, to work in Colab, I use

> !python entry_with_update.py --theme dark --share --always-high-vram
